Tests/Packet++Test/main.cpp

```diff
@@ -250,8 +250,12 @@ int main(int argc, char* argv[])
 	PTF_RUN_TEST(NtpParsingV4Tests, "ntp");
 	PTF_RUN_TEST(NtpCraftingTests, "ntp");
 
-	PTF_RUN_TEST(TelnetCommandParsingTests, "telnet");
+  	PTF_RUN_TEST(TelnetCommandParsingTests, "telnet");
 	PTF_RUN_TEST(TelnetDataParsingTests, "telnet");
 
+	PTF_RUN_TEST(FtpParsingIpv4Tests, "ftp");
+	PTF_RUN_TEST(FtpParsingIpv6Tests, "ftp");

```

Comment:
I'm not sure we need 2 separate tests for IPv4 and IPv6 - there is no difference in parsing the FTP layer itself. The test can include both IPv4  and IPv6 packets